templates/base_layout.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>
+    {% if title %}{{ title }} | MicroStack{% else %}MicroStack - OpenStack in a snap{% endif %}
+  </title>
+
+  <link rel="preconnect" href="https://www.google-analytics.com">
+  <link rel="preconnect" href="https://www.gstatic.com">
+
+  <!-- link rel="stylesheet"-->
+  <link rel="stylesheet" href="{{ versioned_static('css/styles.css') }}" />
+
+  <!-- Google Analytics and Google Optimize -->
+  <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+  ga('create', 'UA-1018242-59', 'auto', {'allowLinker': true});
+  ga('require', 'GTM-WVQJJBD');
+  ga('require', 'linker');
+  ga('linker:autoLink', ['conjure-up.io', 'login.ubuntu.com', 'www.ubuntu.com',
+  'ubuntu.com','cn.ubuntu.com',
+  'design.ubuntu.com', 'maas.io', 'canonical.com', 'landscape.canonical.com',
+  'pages.ubuntu.com', 'tutorials.ubuntu.com', 'docs.ubuntu.com']);
+  </script>
+  <!-- End Google Analytics and Google Optimize -->
+
+  <!-- Google Tag Manager -->
+  <script>(function (w, d, s, l, i) {
+      w[l] = w[l] || []; w[l].push({
+        'gtm.start':
+          new Date().getTime(), event: 'gtm.js'
+      }); var f = d.getElementsByTagName(s)[0],
+        j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
+          'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
+    })(window, document, 'script', 'dataLayer', 'GTM-TDV3PRK');</script>
+  <!-- End Google Tag Manager -->
+
+
+  <link rel="shortcut icon" type="image/png" href="https://assets.ubuntu.com/v1/d53eeeac-MicroK8s_favicon_64px_v2.png">
+
+  <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/e8c07df6-Ubuntu-L_W.woff2" crossorigin>
+  <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/7f100985-Ubuntu-Th_W.woff2" crossorigin>
+  <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/f8097dea-Ubuntu-LI_W.woff2" crossorigin>
+  <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/fff37993-Ubuntu-R_W.woff2" crossorigin>
+
+  <script defer src="{{ versioned_static('js/copy-to-clipboard.js') }}"></script>
```

Comment:
Do we need these 2 scripts on all pages? If not I would load them only on the pages we need them.

Reply:
Done. I have put on the top, because otherwise it would flash if js is not loaded before the rest of the content